pkg/cmd/console.go

```diff
@@ -30,44 +26,6 @@ const (
 	BlueOceanPath = "/blue"
 )
 
-var (
-	console_long = templates.LongDesc(`
-		Opens the Jenkins X console in a browser.`)
-	console_example = templates.Examples(`
-		# Open the Jenkins X console in a browser
-		jx console
-
-		# Print the Jenkins X console URL but do not open a browser
-		jx console -u
-		
-		# Open the Jenkins X console in a browser using the classic skin
-		jx console --classic`)
-)
-
-func NewCmdConsole(commonOpts *opts.CommonOptions) *cobra.Command {
-	options := &ConsoleOptions{
-		GetURLOptions: get.GetURLOptions{
-			GetOptions: get.GetOptions{
-				CommonOptions: commonOpts,
-			},
-		},
-	}
-	cmd := &cobra.Command{
-		Use:     "console",
-		Short:   "Opens the Jenkins console",
-		Long:    console_long,
-		Example: console_example,
-		Run: func(cmd *cobra.Command, args []string) {
-			options.Cmd = cmd
-			options.Args = args
-			err := options.Run()
-			helper.CheckErr(err)
-		},
-	}
-	options.addConsoleFlags(cmd)
-	return cmd
-}
-
 func (o *ConsoleOptions) addConsoleFlags(cmd *cobra.Command) {
```

Comment:
The remaining functions are used in the `jx open` command, maybe the logic here should be moved into that file?

Reply:
By that file I mean https://github.com/jenkins-x/jx/blob/master/pkg/cmd/open.go